RebornCore/src/main/java/reborncore/common/multiblock/rectangular/RectangularMultiblockBlockEntityBase.java

```diff
@@ -121,7 +121,7 @@ public void recalculateOutwardsDirection(BlockPos minCoord, BlockPos maxCoord) {
 		}
 	}
 
-	// /// Validation Helpers (IMultiblockPart)
+	// Validation Helpers (IMultiblockPart)
```

Comment:
this seems like a wrong correction

Reply:
accidentally removed these, then reverted, guess i missed this one